src/components/HomepagePanel/HomepagePanel.tsx

```diff
@@ -104,7 +103,12 @@ export const HomepagePanel = () => {
           <div>
             <Center mb>
               {isClimbing ? (
-                <LogoOpenclimbing width={130} height={130} />
+                <img
+                  src="/openclimbing/logo/osmapp.svg"
```

Comment:
1) I am little confused. Shouldn't this be openclimbing logo?

2) what is the reason to prefer `<img src>` instead of inline svg?

Reply:
1. yes, wrong naming, will fix it
2. we already have this logo because we use it somewhere else and it's preventing duplicity IMHO

Reply:
2. my reason to inline it was, that it is faster to render and doesnt need an additional HTTP request, which is imho importnant (especially) for logo. Also you get the logo from the first SSR response, even if the internet connection falls down 🙂 

btw, the standalone logo `osmapp.svg` wasnt used anywhere in the app, it was only for graphical works.. etc.